tokio/tests/fs_file.rs

```diff
@@ -194,6 +194,9 @@ async fn read_file_from_unix_fd() {
     let raw_fd = file1.as_raw_fd();
     assert!(raw_fd > 0);
 
+    // We will reuse the raw file descriptor.
+    std::mem::forget(file1.into_std().await);
+
```

Comment:
It might be worth to use `file1.into_std().await.into_raw_fd()` instead of forgetting it.

Reply:
Yeah, you're right.